web-app/css/custom.css

```diff
@@ -8880,7 +8880,7 @@ ul.notification-menu > li.see-more {
 .survey-question-answer .only-predefined-option label,
 .survey-question-answer .single-option label,
 .survey-question-answer .multi-option label{
-    line-height: 30px;
+    line-height: 175%;
```

Comment:
I prefer an absolute height. 

Other point, is that the first line is not centered with the bullet icon